lib/ship.rb

```diff
@@ -0,0 +1,32 @@
+require 'pry';
+
+
+class Ship
+
+  attr_reader :ship_name, :length
+  attr_accessor :health
```

Comment:
Not sure if we need attr_accessor.

Reply:
I'm not 100% sure either, but I think if it is a method then we don't because methods are accessible from outside of the class. I guess we will find out once we merge with cell and test it!

Reply:
changed it to an attr_reader and it passed all the test. Havent uploaded the change yet. Doing it soon!